hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -56,6 +68,51 @@ export const getTeamOrders = createAsyncThunk<
     }
 });
 
+export interface PatchOrderResponse {
+    id: number;
```

Comment:
I'm pretty sure this is just the Order type, or u can use parts of the order type here, check types.ts